doc/components.rst

```diff
@@ -5,55 +5,71 @@
 Resources
 =========
 
-SerialPort
-----------
+.. autoclass:: labgrid.resource.RawSerialPort
+   :members:
+   :show-inheritance:
 
-RawSerialPort
--------------
-
-NetworkSerialPort
------------------
+.. autoclass:: labgrid.resource.NetworkSerialPort
+   :members:
+   :show-inheritance:
 
-NetworkService
---------------
+.. autoclass:: labgrid.resource.NetworkService
+   :members:
+   :show-inheritance:
 
-OneWirePIO
-----------
+.. autoclass:: labgrid.resource.OneWirePIO
+   :members:
+   :show-inheritance:
 
-NetworkPowerPort
-----------------
+.. autoclass:: labgrid.resource.NetworkPowerPort
+   :members:
+   :show-inheritance:
 
-RemotePlace
------------
+.. autoclass:: labgrid.resource.RemotePlace
+   :members:
+   :show-inheritance:
 
-USBSerialPort
--------------
+.. autoclass:: labgrid.resource.USBSerialPort
+   :members:
+   :show-inheritance:
 
 Drivers
 =======
 
 CommandProtocol
-------------
+---------------
+.. autoclass:: labgrid.protocol.CommandProtocol
```

Comment:
I started to group the Drivers by Protocol, but since we can just list their base classes I will remove that.